assets/src/blocks/ENForm/ENFormFrontend.js

```diff
@@ -308,6 +280,52 @@ const submitENForm = (props) => {
     });
 }
 
+/**
+ * Build data to be posted on form submit
+ *
+ * @param  {Object}  form_data  The form data
+ * @param  {Array}   fields     The fields
+ * @return {Object}  Formatted data for EN
+ */
+const makePostData = (form_data, fields) => {
+  let supporter = {
+    questions: {}
+  };
+
+  for (const key in form_data) {
+    let field = fields.find((f) => inputId(f)['name'] === key);
+    if ( ! field ) {
+      continue;
+    }
+
+    // Questions via checkbox or text question
+    if ( key.startsWith('supporter.questions.' )) {
+      let value = typeof form_data[key] === "string" ? form_data[key] : checkboxValue(form_data[key]);
```

Comment:
Use `const` instead of `let` since as far as I see, you're not reassigning the value